src/idsk/common.js

```diff
@@ -0,0 +1,16 @@
+/**
+* Toggle class
+* @param {object} node element
+* @param {string} className to toggle
+*/
+export function toggleClass(node, className) {
```

Comment:
Podobnu funkcionalitu si pamatam, ze aj Vilo pouzil pre footer, len taky note, ze mozno sa potom da pouzit tato common funkcia. Aby sa na to ked tak neskor nezabudlo a upravilo

Reply:
dobry postrej :) ano ano viem o tom videl som to a v mergi to resolve-nem :)